code-react/src/views/dashboard/app/user-profile-edit.js

```diff
@@ -44,206 +114,299 @@ const UserProfileEdit =() =>{
-                                              </Form.Check>
-                                              <Form.Check className="form-check d-block">
-                                                  <Form.Check.Input type="checkbox" className="form-check-input" id="email03"defaultChecked/>
-                                                  <Form.Check.Label className="form-check-label" htmlFor="email03">Someone adds you as a connection</Form.Check.Label>
-                                              </Form.Check>
-                                          </Col>
-                                      </Form.Group>
-                                      <Form.Group className="form-group row align-items-center">
-                                          <Form.Label className="col-md-3" htmlFor="npass">When To Escalate Emails</Form.Label>
-                                          <Col md="9">
-                                              <Form.Check className="form-check">
-                                                  <Form.Check.Input className="form-check-input" type="checkbox" defaultValue="" id="email04"/>
-                                                  <Form.Check.Label className="form-check-label"htmlFor="email04">
-                                                      Upon new order.
-                                                  </Form.Check.Label>
-                                              </Form.Check>
-                                              <Form.Check className="form-check d-block">
-                                                  <Form.Check.Input className="form-check-input" type="checkbox" defaultValue="" id="email05"/>
-                                                  <Form.Check.Label className="form-check-label"htmlFor="email05">New membership approval</Form.Check.Label>
-                                              </Form.Check>
-                                              <Form.Check className="form-check d-block">
-                                                  <Form.Check.Input type="checkbox" className="form-check-input" id="email06"defaultChecked/>
-                                                  <Form.Check.Label className="form-check-label"htmlFor="email06">Member registration</Form.Check.Label>
-                                              </Form.Check>
-                                          </Col>
-                                      </Form.Group>
-                                      <Button type="submit" className="btn btn-primary me-2">Submit</Button>
-                                      <Button type="reset" className="btn bg-soft-danger">Cancel</Button>
-                                  </Form>
-                                  </Card.Body>
-                              </Card>
-                          </Tab.Pane>
-                          <Tab.Pane eventKey="fourth" className="fade show">
+    )
+}
+
+
+function FamilyMember({member}){

```

Comment:
Password is misssing here note that the parent can change the password of a family member, Its implemented in the Backend (Image below). Set it as an empty fields and if provided then send it with input 
![image](https://github.com/aldab2/family_space/assets/34516271/7d89aea1-6a9b-4940-b1f8-66d648bf7c22)